lib/generators/solidus_stripe/install/install_generator.rb

```diff
@@ -1,20 +1,37 @@
+# frozen_string_literal: true
+
 module SolidusStripe
   module Generators
     class InstallGenerator < Rails::Generators::Base
+      class_option :migrate, type: :boolean, default: true
       class_option :auto_run_migrations, type: :boolean, default: false
+      class_option :auto_run_seeds, type: :boolean, default: false
+      class_option :stripe_seeds_secret_key, type: :string
```

Comment:
We don’t need to have extra options when we already use env vars in the seeds file.